io.edcd.EDMarketConnector.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: io.edcd.EDMarketConnector
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+command: edmarketconnector
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=~/.steam/steam/steamapps/compatdata/359320/pfx/drive_c/users/steamuser/Saved\ Games/Frontier\ Developments/Elite\ Dangerous
+cleanup:
+  - '/man'
+  - '/include'
+  - '/share/man'
+  - '/lib/pkgconfig'
+modules:
+  - python3-requirements.yaml
+
+  - python3-tkinter.yaml
+
+  - name: edmarketconnector
+    buildsystem: simple
+    build-commands:
+      - mkdir edmarketconnector
+      - tar xzfv edmarketconnector.tar.gz --directory=edmarketconnector/ --strip-components=1
```

Comment:
you could set the source to be `type: archive` instead of extracting by hand.

Reply:
Tracking this in https://github.com/flathub/io.edcd.EDMarketConnector/issues/7